modules/ROOT/pages/enable-verbose-logging.adoc

```diff
@@ -0,0 +1,68 @@
+= Enable Verbose Logging
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can enable verbose logging for different Modules and Connectors. Because of the high impact caused in application performance, verbose logging must be enabled only for testing and troubleshooting purposes.
```

Comment:
Lowercase "modules" and "connectors."

"Because of the high impact caused in application performance" > "Because of the high impact to application performance, limit verbose logging to testing and troubleshooting operations."